modules/nw-ovn-kubernetes-features.adoc

```diff
@@ -5,17 +5,19 @@
 [id="nw-ovn-kubernetes-purpose_{context}"]
 = OVN-Kubernetes purpose
 
-The OVN-Kubernetes network plugin is an open-source, fully-featured Kubernetes CNI plugin that uses Open Virtual Network (OVN) to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution. The OVN-Kubernetes network plugin:
+The OVN-Kubernetes network plugin is an open-source, fully-featured Kubernetes CNI plugin that uses Open Virtual Network (OVN) to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution. The OVN-Kubernetes network plugin uses the following technologies:
 
-// OVN (Open Virtual Network) is consistent with upstream usage.
+* OVN to manage network traffic flows.
+* Kubernetes network policy support and logs, including ingress and egress rules.
+* The Generic Network Virtualization Encapsulation (Geneve) protocol rather than Virtual Extensible LAN (VXLAN) to create an overlay network between nodes.
 
-* Uses OVN (Open Virtual Network) to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution.
-* Implements Kubernetes network policy support, including ingress and egress rules.
-* Uses the Geneve (Generic Network Virtualization Encapsulation) protocol rather than VXLAN to create an overlay network between nodes.
+The OVN-Kubernetes network plugin supports the following capabilities:
 
-The OVN-Kubernetes network plugin provides the following advantages over OpenShift SDN.
-
-* Full support for IPv6 single-stack and IPv4/IPv6 dual-stack networking on supported platforms
-* Support for hybrid clusters with both Linux and Microsoft Windows workloads
-* Optional IPsec encryption of intra-cluster communications
-* Offload of network data processing from host CPU to compatible network cards and data processing units (DPUs)
+* Hybrid clusters that can run both Linux and Microsoft Windows workloads. This environment is known as _hybrid networking_.
+* Offloading of network data processing from the host central processing unit (CPU) to compatible network cards and data processing units (DPUs). This is known as _hardware offloading_.
```

Comment:
Did you see somewhere that we need to define CPU on first use? Seems odd to me in tech docs.

Reply:
I just did it as DPUs is defined and wanted to be consistent in case DPU is also common knowledge to some in a specialized area :smile: .

Reply:
lol yeah I wondered if that was the thinking - it's a bit of an odd case for sure :)